src/main/java/uk/gov/justice/digital/delius/config/SwaggerConfig.java

```diff
@@ -2,40 +2,38 @@
 
 import com.google.common.base.Predicates;
 import com.google.common.collect.ImmutableList;
-import org.springframework.beans.BeansException;
-import org.springframework.beans.factory.annotation.Autowired;
+import io.swagger.util.ReferenceSerializationConfigurer;
+import lombok.AllArgsConstructor;
 import org.springframework.boot.info.BuildProperties;
-import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration;
 import springfox.documentation.builders.RequestHandlerSelectors;
 import springfox.documentation.service.ApiInfo;
 import springfox.documentation.service.Contact;
-import springfox.documentation.service.StringVendorExtension;
-import springfox.documentation.service.VendorExtension;
 import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.json.JacksonModuleRegistrar;
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 import java.time.LocalDateTime;
 import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.Collection;
+import java.util.List;
 import java.util.Optional;
-import java.util.Properties;
 
 import static springfox.documentation.builders.PathSelectors.regex;
 
 @Configuration
 @EnableSwagger2
+@AllArgsConstructor
+@Import(BeanValidatorPluginsConfiguration.class)
 public class SwaggerConfig {
-
-    @Autowired
-    private ApplicationContext applicationContext;
+    private final BuildProperties buildProperties;
```

Comment:
note that switching to this being required will require people to run the app using the gradle task in intellij